functionalDB/chapter.md

```diff
@@ -146,40 +148,37 @@ And here's our in-memory implementation of the protocol, which uses a map as the
    (drop-entity [storage entity] (dissoc storage (:id entity))))
 ````
 
-### Querying our data
+### Querying our Data

```

Comment:
How about making the title of this section "Indexing the data" instead of "Querying our data"?